app/src/main/java/com/techcourse/dao/UserDao.java

```diff
@@ -1,121 +1,52 @@
 package com.techcourse.dao;
 
 import com.techcourse.domain.User;
+import java.util.List;
 import nextstep.jdbc.JdbcTemplate;
+import nextstep.jdbc.RowMapper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.sql.DataSource;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.List;
-
 public class UserDao {
 
     private static final Logger log = LoggerFactory.getLogger(UserDao.class);
 
-    private final DataSource dataSource;
+    private static final RowMapper<User> userRowMapper = resultSet ->
+            new User(
+                    resultSet.getLong(1),
+                    resultSet.getString(2),
+                    resultSet.getString(3),
+                    resultSet.getString(4)
+            );
```

Comment:
c : index로 resultSet을 가져오는 것도 좋지만, 필드명을 명시해주면 더 읽기 좋을 것 같아요!

```suggestion
            new User(
                    resultSet.getLong("id"),
                    resultSet.getString("account"),
                    resultSet.getString("password"),
                    resultSet.getString("email")
            );
```